ctapipe/utils/astro.py

```diff
@@ -0,0 +1,55 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.coordinates import Angle
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+import logging
+
+logger = logging.getLogger(__name__)
+
+__all__ = ['get_bright_stars']
+
+
+def get_bright_stars(pointing=SkyCoord(ra=0. * u.rad, dec=0. * u.rad, frame='icrs'),
+                     radius=180. * u.deg, magnitude_cut=7.96):
+    """
+    Returns an astropy table containing star positions above a given magnitude within
+    a given radius around a position in the sky, using the Yale bright star catalog
+    which needs to be present in the ctapipe-extra package.
+
+    Parameters
+    ----------
+    pointing: astropy Skycoord
+       pointing direction in the sky
+    radius: astropy angular units
+       Radius of the sky region around pointing position. Default: full sky
+    magnitude_cut: float
+        Return only stars above a given magnitude. Default: 7.96 (all entries)
+
+    Returns
+    -------
+    Astropy table:
+       List of all stars after cuts with names, catalog numbers, magnitudes,
+       and coordinates
+    """
+    from astropy.io import fits
+    from astropy.table import Table
+    from ctapipe.utils import get_dataset_path
+
+    catalog = get_dataset_path("yale_brigh2t_star_catalog5.fits.gz")
+    hdulist = fits.open(catalog)
+    table = Table(hdulist[1].data)
+
+    starpositions = SkyCoord(ra=Angle(table['RAJ2000'], unit=u.deg),
+                             dec=Angle(table['DEJ2000'], unit=u.deg), frame='icrs')
+    table['ra_dec'] = starpositions
+
+    if radius < 180. * u.deg:
```

Comment:
I think skycoords have a more efficient way to do this:

https://docs.astropy.org/en/stable/api/astropy.coordinates.SkyCoord.html#astropy.coordinates.SkyCoord.search_around_sky

Reply:
I am not sure, both how to implement it and if it improves much, unless you see a globally better way to implement this feature (as some other object instead of returning an astropy table), there are surely many ways to do it. First, `search_around_sky()` apparently requires an array of SkyCoord's, not a Skycoord object containing many pointings, as the current syntax loads the data (I couldn't find a solution to revert the order). Second, I fear to then produce the cut table also with still at least containing the star names and magnitudes also requires some dedicated lines... So I didn't see a solution on the short term, but if you do, please let me know!

Reply:
Yeah, sorry. Even the error message mentions that:
```
ValueError: One of the inputs to search_around_sky is a scalar. search_around_sky is intended for use with array coordinates, not scalars.  Instead, use ``coord1.separation(coord2) < seplimit`` to find the coordinates near a scalar coordinate```